research/context/preselection/preselect_wikipedia_comments.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tqdm import tqdm
+import pandas as pd
+import tensorflow as tf
+
+# Connect to Google's BigQuery
+from google.cloud import bigquery
+client = bigquery.Client(project="wikidetox-viz")
+
+FLAGS = tf.app.flags.FLAGS
+tf.app.flags.DEFINE_integer("zones_num", 10, "Number of zones to be used for sampling.")
+tf.app.flags.DEFINE_string("output_sample_path", None, "Path where the output sample CSV should be written.")
+tf.app.flags.DEFINE_string("output_short_sample_path", None, "Path where the small (100) output sample CSV should be written.")
+
+def query(low_tox=0, high_tox=1, rand=.5, limit=1000, type="ADDITION", max_txt=1000):
```

Comment:
I think the limit should be a command line argument also. 
And the max_txt too.